modules/webhook/slack.go

```diff
@@ -289,12 +291,11 @@ func getSlackPullRequestApprovalPayload(p *api.PullRequestPayload, slack *SlackM
 func getSlackRepositoryPayload(p *api.RepositoryPayload, slack *SlackMeta) (*SlackPayload, error) {
 	senderLink := SlackLinkFormatter(setting.AppURL+p.Sender.UserName, p.Sender.UserName)
 	repoLink := SlackLinkFormatter(p.Repository.HTMLURL, p.Repository.FullName)
-	var text, title, attachmentText string
+	var text string
 
 	switch p.Action {
 	case api.HookRepoCreated:
 		text = fmt.Sprintf("[%s] Repository created by %s", repoLink, senderLink)
-		title = p.Repository.HTMLURL
```

Comment:
Is this right?

Reply:
@zeripath Yeah, if you look at the images all this used to do was add an attachment with no actual content, just a title with the repo URL (which is already part of `repoLink`)   
It also broke the hook for delete events because title was only ever given a value here.